### PR TITLE
refactor(logging): updated log messages

### PR DESCRIPTION
**Pull request by brianosburn-intel**

### CODE REVIEW

File: src/server/webserver.ts [hunk 1]
```diff
@@ -208,7 +208,7 @@ export class WebServer {
             }
           } else {
             // If TLS is going to be used, setup a TLS socket
-            log.info('TLS Enabled!')
+            log.debug('TLS Enabled!')
```
Comment (rsdmike) [inline src/server/webserver.ts:211]: seems like this one can remain info -- its only every logged at most once during start up right?

File: src/server/webserver.ts [hunk 2]
```diff
@@ -335,7 +335,7 @@ export class WebServer {
         this.relaywss.emit('connection', ws, request)
       })
     } else { // Invalid endpoint
-      log.info('Route does not exist. Closing connection...')
+      log.error('Route does not exist. Closing connection...')
```
Comment (rsdmike) [inline src/server/webserver.ts:338]: this feels like a debug to me. this is a user initiated problem

File: src/utils/certificates.ts [hunk 3]
```diff
@@ -30,7 +30,7 @@ export class certificates {
       rootPrivateKey = fs.readFileSync(rootCertPrivateKeyPath, 'utf8')
       rootCertAndKey = { cert: certoperation.pki.certificateFromPem(rootCertificate), key: certoperation.pki.privateKeyFromPem(rootPrivateKey) }
     } else {
-      log.info('Generating Root certificate...')
+      log.debug('Generating Root certificate...')
```
Comment (rsdmike) [inline src/utils/certificates.ts:33]: eh, kinda like this one as info as well --- one time start up message.

File: src/utils/mqttProvider.ts [hunk 4]
```diff
@@ -69,6 +69,6 @@ export class MqttProvider {
     if (!this.turnedOn) return
 
     this.client = this.client.end()
-    log.info('MQTT client closed')
+    log.debug('MQTT client closed')
```
Comment (rsdmike) [inline src/utils/mqttProvider.ts:72]: info

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: src/server/webserver.ts (hunk 1 above)
Comment (brianosburn-intel) [reply src/server/webserver.ts:211]: yes it looks like it is called in the on connection for the ws

**Review by rsdmike — APPROVED** [review]

**Pull request merged**